web/styles/zulip.css

```diff
@@ -2565,6 +2577,10 @@ div.focused-message-list {
             font-size: 0.8em;
             color: hsl(96deg 7% 73%);
         }
+
+        & a i {
```

Comment:
Again, no element selector. And get rid of as many descendant selectors as you can.